compass/landice/tests/ensemble_generator/plot_ensemble.py

```diff
@@ -182,11 +270,31 @@
     GLX = np.array([])
     GLY = np.array([])
 
+# --------------
+# Get needed fields from input file
+# --------------
+
+DSinput = xr.open_dataset(input_file_path)
+observedSurfaceVelocityX = DSinput['observedSurfaceVelocityX'].values[0, :]
+observedSurfaceVelocityY = DSinput['observedSurfaceVelocityY'].values[0, :]
+obsSpdUnc = DSinput['observedSurfaceVelocityUncertainty'].values[0, :]
+obsSpd = (observedSurfaceVelocityX**2 + observedSurfaceVelocityY**2)**0.5
+obsSpdUnc += obsSpd * 0.25
```

Comment:
Do we actually want this hard-coded?

Reply:
Good catch.  This was an exploration of considering larger speed uncertainty.  I've commented it out with a note.